flower_shop/reports/sale_order_flowers.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+
+        <report
+                id="report_sale_order_flowers_report"
+                model="sale.order"
+                string="Flowers"
+                name="coreline_btco.sale_order_document"
+                file="coreline_btco.sale_order_document"
+                report_type="qweb-pdf"
+                paperformat="coreline_btco.sale_order_paper_format"/>
+
+        <template id="sale_order_document">
+            <t t-call="web.html_container">
+
+                <t t-set="report_header_style" t-value="'display: none;'"/>
+                <t t-set="footer" t-value="'oe_hidden'"/>
+                <t t-call="web.external_layout">
+                    <div class="page">

```

Comment:
Be careful when using the **page** class in reports, as it tells the renderer when to render different pages on different papers.